src/Ubidots.cpp

```diff
@@ -176,9 +176,12 @@ bool Ubidots::wifiConnect(const char *ssid, const char *password) {
   }
 
   WiFi.begin(ssid, password);
+  delay(5000);
```

Comment:
Implement a loop to verify the MAC address when it is available, in order to reduce the amount of time needed to continue the rest of the program.